errors.go

```diff
@@ -1,27 +1,27 @@
 package main
 
 import (
-	"fmt"
-	"log"
 	"net/http"
+
+	"github.com/ONSdigital/log.go/log"
 )
 
 func readFailed(w http.ResponseWriter, err error) {
-	log.Println(fmt.Sprintf("error reading request body: [%s]", err.Error))
+	log.Event(nil, "error reading request body", log.ERROR, log.Error(err))
 	http.Error(w, "cannot read request body", http.StatusInternalServerError)
 }
 
 func bookNotFound(w http.ResponseWriter) {
-	log.Println("book not found in list")
+	log.Event(nil, "book not found in list", log.INFO)
```

Comment:
I'm going to approve the PR, but just wanted to add that you might like to add the requested/failed ID to this log